src/components/invite/useInvites.js

```diff
@@ -0,0 +1,131 @@
+import { useEffect, useState } from 'react'
```

Comment:
1. please add more logging to this fine
2. add .catch to the hooks calling helper methods (e.g. getInviteCode().then().catch() in useInviteCode)
3. let's rename invite**r**Code to some synonym (e.g. 'callerCode') to remove collisions with 'inviteCode'

Reply:
1. switched to using useInviteCode, removed usage of gdstore, using userprops instead
2. added logging + trycatch
3. renamed inviterCode to inviterInviteCode